src/builder.py

```diff
@@ -204,13 +204,15 @@ class CloudImage:
         cloud_id: The cloud ID that the image was uploaded to.
         image_id: The uploaded image ID.
         juju: The juju snap channel.
+        microk8s: The microk8s snap channel.
     """
 
     arch: state.Arch
     base: state.BaseImage
     cloud_id: str
     image_id: str
     juju: str
+    microk8s: str
```

Comment:
Can you remind me if/where these values are validated (that they confirm to the <track>/<risk>/<branch> schema?